src/RadTube/test_radiation_tube.cpp

```diff
@@ -384,12 +384,12 @@ auto problem_main() -> int
 		err_norm += std::abs(Trad_arr[i] - Trad_exact_arr[i]);
 		sol_norm += std::abs(Trad_exact_arr[i]);
 	}
-	for (int i = 0; i < static_cast<int>(xs_exact.size()); ++i) {
-		err_norm += std::abs(Erad_arr_numerical_interp_at_group_1[i] - E1_exact[i]);
-		sol_norm += std::abs(E1_exact[i]);
-		err_norm += std::abs(Erad_arr_numerical_interp_at_group_2[i] - E2_exact[i]);
-		sol_norm += std::abs(E2_exact[i]);
-	}
+	// for (int i = 0; i < static_cast<int>(xs_exact.size()); ++i) {
+	// 	err_norm += std::abs(Erad_arr_numerical_interp_at_group_1[i] - E1_exact[i]);
+	// 	sol_norm += std::abs(E1_exact[i]);
+	// 	err_norm += std::abs(Erad_arr_numerical_interp_at_group_2[i] - E2_exact[i]);
+	// 	sol_norm += std::abs(E2_exact[i]);
+	// }
```

Comment:
Is there any reason to keep this commented-out code?

Reply:
Yes, I'll keep them for future use. This check of E_r for individual bins is compromised because I precalculated E1 and E2 based on numerical results from one run. I will change it to use the analytical solution later.